files/uk/web/html/element/section/index.md

```diff
@@ -0,0 +1,186 @@
+
+{{EmbedInteractiveExample("pages/tabbed/section.html", "tabbed-standard")}}
+
+## Атрибути
+
+Цей елемент приймає лише [глобальні атрибути](/uk/docs/Web/HTML/Global_attributes).
+
+## Примітки щодо використання
+
+Як згадувалось вище, `<section>` є узагальненим розділовим елементом, і його слід використовувати лише тоді, коли немає більш конкретного елемента для представлення вмісту. Наприклад, глобальне меню навігації слід загортати в елемент {{htmlelement("nav")}}, але список результатів пошуку або виведення карти та її елементи управління не мають конкретних відповідних елементів і можуть бути поміщені всередину `<section>`.
+
+Також слід розглянути такі випадки:
+
+- Якщо вміст елемента є самостійною, неподільною одиницею вмісту, яку є сенс синдикувати як самостійний твір (наприклад, допис у блозі чи коментар у блозі, або стаття у газеті), краще використовувати елемент {{HTMLElement("article")}}.
+- Якщо цей вміст представляє додаткову інформацію, яка корисна поруч з основним вмістом, але не є його безпосередньою частиною (наприклад, пов'язані посилання або біографія автора), використовуйте {{HTMLElement("aside")}}.
+- Якщо вміст представляє головну область вмісту документа, використовуйте {{HTMLElement("main")}}.
+- Якщо елемент використовується суто як обгортка для стилізації, використовуйте краще {{HTMLElement("div")}}.
+
+Іще одне повторення: кожен `<section>` повинен бути підписаний, зазвичай шляхом додавання заголовка (елемент {{HTMLElement("Heading_Elements", "h1")}} – {{HTMLElement("Heading_Elements", "h6")}}) як дочірнього щодо `<section>` елемента, де це можливо. Нижче дивіться приклади того, де можна зустріти `<section>` без заголовка.
+
+## Приклади
+
+### Приклад простого застосування
+
+#### До
+
+```html
+<div>
+  <h2>Заголовок</h2>
+  <p>Купа крутого вмісту</p>
+</div>
+```
+
+##### Результат
+
+{{EmbedLiveSample('do')}}
+
+#### Після
+
+```html
+<section>
+  <h2>Заголовок</h2>
+  <p>Купа крутого вмісту</p>
+</section>
+```
+
+##### Результат
+
+{{EmbedLiveSample('pislia')}}
+
+### Використання розділу без заголовка
+
+Умови, за яких можна зустріти `<section>` без заголовка, зазвичай трапляються в розділах вебзастосунків або інтерфейсів користувача, а не в структурах традиційних документів. У документі не дуже логічно мати окремий розділ вмісту без заголовка, що описує його вміст. Такі заголовки корисні для всіх читачів, але особливо корисні для користувачів допоміжних технологій, таких як читачі з екрана, а також вони добрі для SEO.
+
+Проте розгляньмо другорядний механізм навігації. Якщо глобальна навігація вже загорнута в елемент `<nav>`, то можна загорнути меню переходу назад і далі в `<section>`:
+
+```html
+<section>
+  <a href="#">Попередня стаття</a>
+  <a href="#">Наступна стаття</a>
+</section>
+```
+
+Або що на рахунок панелі кнопок для керування застосунком? Тоді не обов'язково потрібен заголовок, але це все одно окремий розділ документа:
```

Comment:
```suggestion
Або як щодо панелі кнопок для керування застосунком? Тоді не обов'язково потрібен заголовок, але це все одно окремий розділ документа:
```
https://onlinecorrector.com.ua/%D1%89%D0%BE%D0%B4%D0%BE-%D0%BD%D0%B0-%D1%80%D0%B0%D1%85%D1%83%D0%BD%D0%BE%D0%BA/